books/RayTracingInOneWeekend.html

```diff
@@ -920,15 +920,15 @@
     class hittable_list: public hittable {
         public:
             hittable_list() {}
-            hittable_list(hittable* object) { add(object); }
+            hittable_list(shared_ptr<hittable> object) { add(object); }
```

Comment:
Needs
```
#include <memory>
```

Reply:
Memory is a project-wide include, via `rtweekend.h`. (We include cstdlib, limits, cmath and memory.)

Reply:
Ah, but `rtweekend.h` is presented _after_ `hittable_list`. Sigh. I'll rework the text.